src/main/java/seedu/address/commons/util/StringUtil.java

```diff
@@ -38,6 +38,27 @@ public static boolean containsWordIgnoreCase(String sentence, String word) {
                 .anyMatch(preppedWord::equalsIgnoreCase);
     }
 
+    /**
+     * Returns true if the {@code sentence} contains the {@code word}.
+     *   Ignores case, and a full match is not required.
+     *   <br>examples:<pre>
+     *       containsSubstringIgnoreCase("ABc def", "abc") == true
+     *       containsSubstringIgnoreCase("ABc def", "DEF") == true
+     *       containsSubstringIgnoreCase("ABc def", "AB") == true
+     *       </pre>
+     * @param sentence cannot be null
+     * @param subString cannot be null, cannot be empty
+     */
+    public static boolean containsSubstringIgnoreCase(String sentence, String subString) {
+        requireNonNull(sentence);
+        requireNonNull(subString);
+
+        String preppedString = subString.trim();
+        checkArgument(!preppedString.isEmpty(), "Substring parameter cannot be empty");
+
+        return sentence.toLowerCase().contains(preppedString.toLowerCase());
+    }
+
```

Comment:
Nice job creating a function to see if a substrings exists in a string! Our functionality of find command would be much better. I see that it is has good JUnit tests as well